src/lib/util.ts

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+import {utils} from '@heroku/heroku-cli-util'
+
+// Using the same type pattern as in bloat.ts
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Database = any
+
+interface Plan {
+  plan: {
+    name: string
+  }
+}
```

Comment:
We should use `Heroku.AddOn['plan']` for this. Add `import * as Heroku from '@heroku-cli/schema'` to the imports, that's where we house our types for the Heroku API.